typescript/commands/build.py

```diff
@@ -38,3 +40,4 @@ def compile_inferred_project(self, file_name, params=""):
             "cmd": cmd,
             "file_regex": "^(.+?)\\((\\d+),(\\d+)\\): (.+)$"
         })
+        self.build_parameters = params
```

Comment:
I think empty string as <code>params=""</code> means the default value of <code>params</code> is empty string.